person_capture/gui_app.py

```
@@ -4299,6 +4740,15 @@ def _apply_live_updates() -> None:
                                     fx2i = int(round(float(np.clip(fx2 + off_x, 0, W - 1))))
                                     fy2i = int(round(float(np.clip(fy2 + off_y, 0, H - 1))))
                                     face_box_global = (fx1i, fy1i, fx2i, fy2i)
+                                    subject_box_global = None
+                                    if subject_box_abs is not None:
+                                        sx1, sy1, sx2, sy2 = subject_box_abs
+                                        subject_box_global = (
+                                            max(0, min(W - 1, int(round(sx1 + off_x)))),
```

Ruff (0.15.11)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)